Insights/InputEntity_Host/EventLogClear.yaml

```diff
@@ -78,7 +78,7 @@ BaseQuery: |
   HostClearedEventLog('{{Host_HostName}}', '{{Host_NTDomain}}', '{{Host_DnsDomain}}', '{{Host_AzureID}}', '{{Host_OMSAgentID}}')
 Insights:
  Id: 9a70a72d-25d4-7212-b73e-4f302a90c06a
- DisplayName: Event Logs Cleared on Host
+ DisplayName: Event Logs cleared on host
```

Comment:
Is "Logs" supposed to be capitalized?

Reply:
The UX in Event Viewer has it capitalized and also [Win32 API references](https://docs.microsoft.com/en-us/windows/win32/wes/windows-event-log-reference), so we will leave it.